cluster.yml

```diff
@@ -40,7 +40,27 @@
     - { role: kubespray-defaults }
     - { role: bootstrap-os, tags: bootstrap-os}
 
+- name: Gather facts
+  hosts: all
```

Comment:
People might have inventories with extra groups not related. Please limit to just our groups: k8s-cluster:etcd:calico-rr